company-ycmd.el

```diff
@@ -406,12 +406,12 @@ candidates list."
   (let* ((prefix-start-col (- (+ 1 (ycmd--column-in-bytes)) (length prefix)))
          (prefix-size (- start-col prefix-start-col))
          (prefix-diff (substring-no-properties prefix 0 prefix-size))
+         (prefix-diff-p (s-present? prefix-diff))
```

Comment:
@abingham Can you remember of an use-case where `prefix-diff` is `non-nil`? Would be good to add a unit-test for it.